markdown/en/core-result-dispatcher.md

```diff
@@ -4,7 +4,7 @@ subscribe to ResultDispatcher to get notified when new data is available.
 
 The ResultDispatcher is in the module: **factfinder.communication.ResultDispatcher**.
 
-If you want to trigger your own communication events checkout the documentation for the [FFCommunicationEventAggregator](documentation/core-event-aggregator).
+If you want to trigger your own communication events checkout the documentation for the [FFCommunicationEventAggregator](api/core-event-aggregator).
```

Comment:
I would make it `api/core-event-aggregator#tab=docs` to the link. When the fragment is added by the app, another entry in the browser history is created and you have to click twice to navigate back. (At least in my Chrome)